crates/core/tedge_mapper/src/core/mapper.rs

```diff
@@ -86,7 +87,7 @@ impl Mapper {
         }
     }
 
-    pub(crate) async fn run(&mut self, ops_dir: Option<PathBuf>) -> Result<(), MqttError> {
+    pub(crate) async fn run(&mut self, ops_dir: Option<&Path>) -> Result<(), MqttError> {
```

Comment:
This change seems to be unrelated to the scope of this PR, or am I missing something?